site/gdocs/OwidGdocPage.tsx

```diff
@@ -146,6 +151,59 @@ function JsonLdArticle({
     )
 }
 
+function JsonLdProfilePage({
+    gdoc,
+    baseUrl,
+    imageUrl,
+}: {
+    gdoc: OwidGdocAuthorInterface
+    baseUrl: string
+    imageUrl?: string
+}) {
+    const mainAuthorId = `#${gdoc.slug}`
+    const data: WithContext<ProfilePage> = {
+        "@context": "https://schema.org",
+        "@type": "ProfilePage",
+        // NOTE: We don't set dateModified to gdoc.updatedAt, because the
+        // semantics of these fields is different. gdoc.updatedAt is the time
+        // the gdoc row was updated in the database, even if the content hasn't
+        // changed and can be even earlier than gdoc.publishedAt for articles
+        // scheduled for publication into the future.
+        dateCreated: gdoc.publishedAt?.toISOString(),
+        mainEntity: {
+            "@id": mainAuthorId,
+            "@type": "Person",
+            name: gdoc.content.title,
+            jobTitle: gdoc.content.role,
+            description: gdoc.content.bio
+                ?.map((block) => spansToUnformattedPlainText(block.value))
+                .join(" "),
+            image: imageUrl,
+            url: getCanonicalUrl(baseUrl, gdoc),
+        },
+        hasPart: gdoc.latestWorkLinks?.slice(0, 10).map((work) => {
+            return {
+                "@type": "Article",
+                headline: work.title,
+                url: getCanonicalUrl(baseUrl, {
+                    slug: work.slug,
+                    content: { type: OwidGdocType.Article },
+                }),
+                datePublished: work.publishedAt || undefined,
```

Comment:
Is it possible for something here to not be published? Shouldn't we filter those out in that case?